### DISCUSSION TITLE
Is it possible SBFSpot for ESP32?

### DISCUSSION THREAD

**Discussion by AingTii** [SBFspot/SBFspot#487] [Ideas]
Hello,
@sbf- 

I have just an Idea, may it be possible to have a very simple sbfspot for an esp32?

I searched already how to connect a ESP32 via bluetooth classic but only find
how to connect smartphone app with esp32 but not how to connect from esp32 to an device
like SMA inverter or just an other esp32.

Do you have expirences with ESP32 bluetooth classic?
may it be possible to connect to an Inverter to get the data and send it via wifi to iobroker etc...
maybe via mqtt....

thank you
Best Regards
Jens

**Comment by TBR-BRD** [comment]
cool idea, im searching for a esp32 to pvoutput.org device for 2 Bluetooth connected inverters.

**Comment by sbf-** [comment]
This can be a good starting point: [Nanode SMA PV monitor](https://github.com/stuartpittaway/nanodesmapvmonitor)

**Reply by AingTii** [reply]
ok, in that way, I have to look how it works with esp32 and if its is able to work with AT commands in the ESP32
